recipes/opencolorio/all/patches/1.1.1.patch

```diff
@@ -72,7 +136,8 @@ index e4f3119..1f7f77c 100644
 -    if(YAML_CPP_VERSION VERSION_LESS ${YAML_VERSION_MIN})
 -        message(FATAL_ERROR "ERROR: yaml-cpp ${YAML_VERSION_MIN} or greater is required.")
 -    endif()
--
++find_package(yaml-cpp)
```

Comment:
Why doesn't it have `REQUIRED`?
If this is the way to switch between addition or removal of yaml support, then perhaps the recipe should also use 
[`CMAKE_DISABLE_FIND_PACKAGE_yaml-cpp`](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html).

Reply:
Yep, it should have `REQUIRED` as it is not optional. This is a remnant of the original patch, but fixed it now. Also added `REQUIRED` to lcms `find_package` as well.